app/api-v1/routes/attachment/{id}.js

```diff
@@ -94,6 +47,9 @@ module.exports = function (attachmentService) {
                   type: 'array',
                   items: {},
                 },
+                {
+                  type: 'string',
+                },
```

Comment:
Why have we added this?

Reply:
accident, leftover from buildValiudator, it has been removed